src/ucp/core/ucp_request.c

```diff
@@ -60,6 +60,14 @@ ucp_request_release_common(void *request, uint8_t cb_flag, const char *debug_nam
     UCP_THREAD_CS_EXIT_CONDITIONAL(&worker->mt_lock);
 }
 
+void ucp_request_get_cb_data(void *request, void **cb_data)
```

Comment:
They will probably want this as an inline somewhere.

Reply:
This is an API function. Not sure how I would inline this.